docs/learning/guidelines/release-tags-guidelines.md

```diff
@@ -165,22 +50,11 @@ Likewise, *deprecated* API items should include documentation that indicates wha
 ```ts
 /** Original documentation comment is typically maintained here.
  * @see Other public API item that third parties should use instead.
- * @deprecated Comment describing reason API item is deprecated and what should be done instead.
- */
-```
-
-*Beta* and *alpha* API items should include a short reason for that classification.
-
-```ts
-/** Sample description of a beta API item.
- * @beta Comment describing reason API item is beta that will be included in the public SDK documentation.
+ * @public
+ * @deprecated Comment describing reason API item is deprecated and what should be used instead.
```

Comment:
In the "API Categories" it is stated that "deprecated tag is typically accompanied by information about the package version in which it became deprecated" - this should be noted here as well I suppose.

Reply:
True. It's currently an aspirational lie though - nobody has ever actually included version information in a `@deprecated` tag. @aruniverse suggested we start doing that. Perhaps we need to write a script that will add "3.x" to anything currently tagged as deprecated.